packages/prisma-client/prisma/schema.prisma

```diff
@@ -22,6 +22,11 @@ enum Location {
   REMOTE
 }
 
+enum UploadStatus {
```

Comment:
Do we name enums as CAPITALIZED?
Just one enum exists hard to decide

Reply:
prisma uses capitalized, so I would do the same, since I have no opinion on this